atef/report.py

```diff
@@ -349,15 +349,18 @@ def build_data_table(
     story.append(Paragraph('Observed Data', l0))
     # use cached value.
     observed_value = getattr(comp, 'data', 'N/A')
-    if not observed_value:
+    if observed_value is None:
         # attr can be set to None
         observed_value = 'N/A'
     observed_value = Paragraph(str(observed_value), styles['BodyText'])
     try:
-        timestamp = comp.signal.timestamp.ctime()
-        source = Paragraph(getattr(comp.signal, 'name', ''), styles['BodyText'])
+        timestamp = datetime.fromtimestamp(comp.signal.timestamp).ctime()
```

Comment:
Just slightly curious: what does running this through datetime first do?

Reply:
Memory fails me, but I think the timestamp may not be a datetime object in some cases, and could just be an integer.  I wanted to use the `.ctime()` method I guess.